packages/frontend/src/components/connect-button/account-menu/styles.module.css

```diff
@@ -11,7 +11,11 @@
 }
 
 .accountContainer {
-    @apply flex items-center gap-3;
+    @apply flex items-center gap-3 hover:bg-gray-200 p-2 rounded-md cursor-pointer;
```

Comment:
We should add the usual `duration-200 transition-colors ease-in-out` when we change styles on user interactions.